Week1/homework.md

```diff
@@ -0,0 +1,52 @@
+2. Assignment
```

Comment:
Well done Alaa! I like your wide thinking and how you try to cover as much as you can of test activities:)

Reply:
thank you @Elias011  for your feedback ,i have added the requested changes.

> Well done Alaa! I like your wide thinking and how you try to cover as much as you can of test activities:)